sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/servicebus/AzureServiceBusProperties.java

```diff
@@ -16,6 +18,8 @@
 
     private String connectionString;
 
+    private AmqpRetryOptions retryOptions = new AmqpRetryOptions().setTryTimeout(ServiceBusConstants.OPERATION_TIMEOUT);
```

Comment:
Is this default value for servicebus sdk client? If so, seems we don't set it here